### PR TITLE
Change chatrix-popup-container zIndex

### LINKED ISSUE #206: chatrix-popup-container gets hidden under some designs.
Tested Chatrix with a custom theme(p2020) and the popup gets hidden under the site elements.

Adding `z-index: 1` to `chatrix-popup-container` seems to fix it at least for the P2 theme.

### PR DESCRIPTION
**Pull request by lezama**
fixes #206

### CODE REVIEW

**Review by ashfame — APPROVED** [review]

**Comment by ashfame** [comment]
Thanks @lezama for the fix!

**Review by psrpinto — APPROVED** [review]

File: src/Popup/popup.js [hunk 1]
```diff
@@ -8,6 +8,7 @@
 		const container = document.createElement("div");
 		container.id = "chatrix-popup-container";
 		document.body.appendChild(container);
+        container.style.zIndex = 1;
```
Comment (ashfame) [inline src/Popup/popup.js:11]: @psrpinto space vs tab bit here:

```
diff --git a/src/Popup/popup.js b/src/Popup/popup.js
index fe69cd2..fb180e8 100644
--- a/src/Popup/popup.js
+++ b/src/Popup/popup.js
@@ -8,6 +8,7 @@
                const container = document.createElement("div");
                container.id = "chatrix-popup-container";
                document.body.appendChild(container);
+        container.style.zIndex = 1;

                Chatrix.renderPopup(container, {
                        defaultHomeserver: config.defaultHomeserver,
```

Leave it as is or fix it? I thought Github used to show the difference 🤔

Comment (psrpinto) [reply src/Popup/popup.js:11]: I guess ideally we would set up a linter for js under `src/`. Could also just fix this one instance for now, I can fix.

Comment (psrpinto) [reply src/Popup/popup.js:11]: https://github.com/Automattic/chatrix/pull/210

**Pull request merged**